playwright/async_api.py

```diff
@@ -4922,10 +4924,11 @@ async def setGeolocation(self, geolocation: typing.Dict) -> NoneType:
 
         Parameters
         ----------
-        geolocation : Optional[typing.Dict]
```

Comment:
Let's try restoring parameter docs here?

Reply:
good catch. Added a fallback for such cases.